politeiawww/database/localdb/localdb.go

```diff
@@ -42,7 +49,7 @@ type Version struct {
 // and false otherwise. This is helpful when iterating the user records
 // because the DB contains some non-user records.
 func isUserRecord(key string) bool {
-	return key != UserVersionKey && key != LastPaywallAddressIndex
+	return key != UserVersionKey && key != LastPaywallAddressIndex && key != AccessTimePrefixKey
```

Comment:
Here you need to check if the key has the prefix: 
```suggestion
	return key != UserVersionKey && key != LastPaywallAddressIndex && !strings.hasPrefix(key, AccessTimePrefixKey)
```

Reply:
fixed